sample-dcrdata.conf

```diff
@@ -50,7 +50,9 @@
 ;pg=true
 
 ; PostgreSQL database config (when pg=true)
-;pgdbname=dcrdata
+; Create a database for each network you plan to run.
+; {netname} will be replaced with the current network name. (e.g. MainNet, TestNet, SimNet)
+;pgdbname=dcrdata_{netname}
```

Comment:
Since the default is still `pgdbname=dcrdata` (and I think it should be), we should leave `;pgdbname=dcrdata` above these new lines and note that it is the default, then on these lines, we would explicitly mention the special template processing.